### PR TITLE
@broskoski - refactor for header slideshow

### PR DESCRIPTION
**Pull request by xtina-starr**

### CODE REVIEW

File: components/header/index.styl [hunk 1]
```diff
@@ -1,77 +1,106 @@
...
+  background-image url('./images/banner-2.jpg')
+
+.slide-3
+  background-image url('./images/banner-3.jpg')
+
+.header-content
+  height 100%
+  display flex
+  flex-direction column
+  justify-content space-around
```
Comment (broskoski) [inline components/header/index.styl:41]: I don't think this is doing anything anymore, probably safe to remove

File: components/header/index.styl [hunk 2]
```diff
@@ -1,77 +1,106 @@
...
+  display none
+
 // mobile
 @media (max-width 320px)
   .header-text > p
-    font-size 15px
+    font-size 13px
 
-@media (max-width 480px)
+@media (orientation: landscape)
```
Comment (broskoski) [inline components/header/index.styl:68]: This is the reason you had to override the header styling with `!important`, you can apply multiple properties to media queries to specifically target phones in landscape mode. For example:

```
@media only screen 
  and (min-device-width: 320px) 
  and (max-device-width: 568px)
  and (-webkit-min-device-pixel-ratio: 2)
  and (orientation: landscape)
```

File: components/header/index.styl [hunk 3]
```diff
@@ -1,77 +1,106 @@
...
+  flex-direction column
+  justify-content space-around
+  z-index 501
+  position absolute
+  top 0
+
+.header-text
+  width 85%
+  margin 10% auto
+  padding-top 30px !important
```
Comment (broskoski) [inline components/header/index.styl:49]: We talked offline about the better way to vertically position this guy (display table method).

**Comment by broskoski** [comment]
Apart from stuff we talked about, its looking good! I think that header refactor definitely went a long way towards reorganizing things :+1:

File: components/header/index.styl [hunk 4]
```diff
@@ -1,77 +1,106 @@
...
+.header-slide
+  width 100%
+  height 100%
+  background-position center center
+  background-size cover
+  -webkit-transition: opacity 1s ease-in
+  -moz-transition opacity 1s ease-in
+  -o-transition opacity 1s ease-in
+  -ms-transition opacity 1s ease-in
+  transition opacity 1s ease-in
```
Comment (dzucconi) [inline components/header/index.styl:18]: http://nibstyl.us/docs/#transparent-mixins

**Comment by broskoski** [comment]
Will merge now, feel free to follow up with refactors later.

**Pull request merged**